src/swap/utils.ts

```diff
@@ -152,41 +156,66 @@ export async function getFixedOutputSwapQuote({
   return getBestQuote(validQuotes);
 }
 
+/**
+ * Returns the rate of a swap quote.
+ *
+ * @param quote - The quote.
+ * @returns The rate.
+ */
+function getSwapQuoteRate(quote: SwapQuote): number {
```

Comment:
```suggestion
/**
 * @returns The asset amount ratio for the given quote
 */
function getSwapQuoteRate(quote: SwapQuote): number {
```
Long comments are hard to maintain, I think we can simplify, or even remove it altogether. I believe function name and param name is enough to understand the function